integration/test/performance_tracker/performance_query_utils.go

```diff
@@ -0,0 +1,143 @@
+package data_collector
+
+import (
+	"time"
+	"errors"
+	"context"
+	"encoding/json"
+	"os"
+	"fmt"
+	
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+)
+
+const (
+	Namespace = "CWAgent"
+	DimensionName = "InstanceId"
+	Stat = "Average"
+	Period = 30
+)
+// GenerateGetMetricInputStruct generates the struct required to make a query request to cloudwatch's GetMetrics
+func GenerateGetMetricInputStruct(ids, metricNames []string, instanceId string, timeDiff int) (*cloudwatch.GetMetricDataInput, error) {
```

Comment:
From what I can tell, this is using GetMetricData and #497 is expecting that. I think the end result will make more sense to call GetMetricStatistics here, but it's okay to have this as is for now while you both explore how interacting with GetMetricStatistics would work. It's possible I'm just wrong and GetMetricStatistics doesn't help at all

Reply:
I will look more into GetMetricStatistics